console/v1/types_console_plugin.go

```diff
@@ -101,7 +101,7 @@ type ConsolePluginSpec struct {
 
 // DirectiveType is an enumeration of OpenShift web console supported CSP directives.
 // LoadType is an enumeration of i18n loading types.
-// +kubebuilder:validation:Enum:="DefaultSrc";"ScriptSrc";"StyleSrc";"ImgSrc";"FontSrc";"ObjectSrc";"ConnectSrc"
+// +kubebuilder:validation:Enum:="DefaultSrc";"ScriptSrc";"StyleSrc";"ImgSrc";"FontSrc";"ConnectSrc"
```

Comment:
When did this object ship? Removing a value from an enum is technically a breaking change, has this shipped only in 4.19?

Reply:
it was added just recently, [last week](https://github.com/openshift/api/pull/2163)